test/posts.test.js

```diff
@@ -166,4 +166,13 @@ describe('test /posts/:id responses', () => {
     expect(res.get('Content-type')).toContain('application/json');
     expect(res.body).toEqual(JSON.parse(JSON.stringify(receivedPost1)));
   });
+
+  test('requests ID length Test', async () => {
+    const res = await request(app)
+      .get(`/posts/${addedPost1.id.slice(0, 4)}`)
+      .set('Accept', 'text/html');
+    expect(res.status).toEqual(400);
+    expect(res.get('content-length')).toContain('34');
+    //expect(res.text).toEqual(receivedPost1.html);
```

Comment:
One of your tests should look something similar to this
 ```
test('should return 200 if it is valid', async () => {
    const id = <A valid id>;
    const res = await request(app).get(`/posts/${id}`);

   \\ Do test expecting here.
    expect(res.status).toEqual(200);
  });
```
I commented on the issue page with more details

Reply:
Now I have, 
1. request with over 10 characters scenario
2. request with less 10 characters scenario

I will keep thinking more case like you mentioned in issue. :)
Thanks!